### PR TITLE
Add "./manage.py check" to travis run

### PR DESCRIPTION
**Pull request by willkg**
r?

### CODE REVIEW

**Comment by mythmon** [comment]
In SUMO, we have a check in the install and run scripts that skip over a lot of the set up when TEST_SUITE=lint to make the tests go faster. Do you want to do something like that here? You'd probably not be able to skip installing Python dependencies. But maybe you could skip installing ES and starting services.

File: bin/travis/dispatch.sh [hunk 1]
```diff
@@ -3,12 +3,11 @@
...
 
 case $SUITE in
-    all )
-        bin/travis/test.sh
-        ;;
     django )
         bin/travis/test.sh
         ;;
+    djangotest )
+        bin/travis/djangocheck.sh
```
Comment (mythmon) [inline bin/travis/dispatch.sh]: Oh, you need to have a `;;` after this line. It is like `break` in other languages' case statements. Apparently not having it is a syntax error here (according to Travis)

Comment (willkg) [inline bin/travis/dispatch.sh]: Bah. Copy/paste fail.

File: bin/travis/install.sh [hunk 2]
```diff
@@ -12,6 +12,10 @@ echo "Install Python dependencies"
 pip freeze
 echo
 
+if [[ $TEST_SETUP == "minimal" ]]; then
```
Comment (mythmon) [inline bin/travis/install.sh]: You'll need a similar check in setup.sh

**Comment by willkg** [comment]
^^^ That fixed all the issues and now passes travis.

**Comment by mythmon** [comment]
LGTM r+

**Pull request merged**